MapboxNavigation/NavigationViewController.swift

```diff
@@ -386,16 +386,16 @@ open class NavigationViewController: UIViewController {
     
     override open func viewDidLoad() {
         super.viewDidLoad()
+        // Initialize voice controller if it hasn't been overridden.
+        // This is optional and lazy so it can be mutated by the developer after init.
```

Comment:
This still feels icky to me. I'd like us to continue to iterate on this in an upcoming PR